lib/Controller/ChatController.php

```diff
@@ -841,12 +850,17 @@ public function getObjectsSharedInRoom(string $objectType, int $lastKnownMessage
 		return $response;
 	}
 
-	protected function getMessagesForRoom(Room $room, array $messageIds): array {
+	private function getMessagesForRoom(Room $room, array $messageIds): array {
```

Comment:
```suggestion
	protected function getMessagesForRoom(Room $room, array $messageIds): array {
```
Please leave all methods and members always protected unless it is really necessary to make something private.